server/db/seed.js

```diff
@@ -102,17 +102,29 @@ const homePhotos = [
   'Singapore/photo-1552406712-1ff867fb0b6e.jpeg',
 ];
 
-const randomNumber = () => {
-  const numbers = [0, 0.5, 1.0, 1.5, 2.0, 2.5, 3.0, 3.5, 4.0, 4.5, 5.0];
-  return numbers[Math.floor(Math.random() * 10)];
+const randomRating = () => {
+  const rating = [0, 0.5, 1.0, 1.5, 2.0, 2.5, 3.0, 3.5, 4.0, 4.5, 5.0];
+  return rating[Math.floor(Math.random() * 10)];
```

Comment:
As an extremely minor point, I would consider either Math.ceil, or extending the multiplied value to 10.4ish, as the only possible way to get a 5.0 rating currently would be to have Math.random return a perfect 1 (1/100 chance, vs. other ratings being higher). The irony of the fact that this is a front-end exercise and somewhat irrelevant is not lost on me :)

Reply:
Interesting, I didn't realize that, thank you!